apps/admin_panel/assets/src/services/currentUserService.js

```diff
@@ -13,9 +15,12 @@ export function getCurrentUserAccount () {
   })
 }
 
-export function updateCurrentUser ({ email }) {
+export function updateCurrentUserEmail ({ email }) {
   return authenticatedRequest({
-    path: 'me.update',
-    data: { email }
+    path: 'me.update_email',
+    data: {
+      email,
+      redirect_url: `${ADMIN_API_URL}verify_email?email={email}&token={token}`
```

Comment:
`ADMIN_API_URL` is an API url, we should use something like
`root_domain/verify_email?email={email}&token={token}`

for Example, 
```js
 redirectUrl: window.location.href.replace(
          this.props.location.pathname,
          '/create-new-password/'
```

Reply:
ref: `ewallet/apps/admin_panel/assets/src/omg-forgetPassword-form/index.js`